src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+/* eslint-disable arrow-body-style */
+const InitialState = [];
+
+export const checkStatus = () => {
+  return {
+    type: 'STATUS_CHECKED',
+  };
+};
```

Comment:
- [ ] Kindly make sure that the type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux).